src/components/symptom-shark/presentational/SymptomTreatmentFilters/SymptomTreatmentFilters.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import { Layout, Section } from "../../../presentational";
+import { previewConfiguration } from "../../container/LogToday/LogToday.previewData";
+import SymptomTreatmentFilters from "./SymptomTreatmentFilters";
+
+export default { title: "SymptomShark/Presentational/SymptomTreatmentFilters", component: SymptomTreatmentFilters, parameters: { layout: 'fullscreen' } };
+
+export function Default() {
+    const [selectedSymptoms, setSelectedSymptom] = useState<string[]>([]);
+    const [selectedTreatments, setSelectedTreatments] = useState<string[]>([]);
+    const [expandedDropdown, setExpandedDropdown] = useState<"Symptoms" | "Treatments" | null>(null);
+
+    return <Layout colorScheme="auto">
```

Comment:
I see the `colorScheme="auto"` in some stories, but not others.  Is that just because we occasionally forget to include it, or is there a rhyme and reason for when you have been including it vs not?

Reply:
Just forgetting to include it